### ISSUE TITLE
 Could pangolin predict the splice site ab initio?

### ISSUE THREAD

**Issue by permia** [tkzeng/Pangolin#10]
Hi, I am trying to predit the splice site of an RNA virus artificially expressed in host genome. Because the organism I studied is not model organism, there is no VCF file. The pangolin seem to must use a vcf file as the positional arguments. Could pangolin predict the splice site ab initio?

**Comment by tkzeng** [comment]
Yes, please see this script: [https://github.com/tkzeng/Pangolin/blob/main/scripts/custom_usage.py](https://github.com/tkzeng/Pangolin/blob/main/scripts/custom_usage.py)

You will need to obtain the sequence around the sites you are interested in predicting. For this use case, you can take the mean or maximum predictions over all tissues.
